.lintr

```diff
@@ -1,5 +1,5 @@
 linters: linters_with_defaults(
-  cyclocomp_linter(complexity_limit = 18),
+  cyclocomp_linter(complexity_limit = 25),
```

Comment:
Hey @sophie-gem this is where we can adjust complexity lintr.  Just FYI most packages have a `.lintr` file to adjust things like this